cmd/meroxa/root/apps/init.go

```diff
@@ -148,7 +148,7 @@ func (i *Init) Execute(ctx context.Context) error {
 		return err
 	}
 
-	i.logger.Infof(ctx, "Application successfully initialized!\n"+
+	i.logger.Infof(ctx, "Turbine Application successfully initialized!\n"+
```

Comment:
Do you prefer to call it `Turbine Data Application` in this instance as well?

```suggestion
	i.logger.Infof(ctx, "Turbine Data Application successfully initialized!\n"+
```

Reply:
@jayjayjpg I thought of it and wasn't sure. This works for me anyways!